code/modules/jobs/job_types/cook.dm

```diff
@@ -88,6 +88,17 @@
 		if(!visualsOnly)
 			other_chefs.cooks++
 
+/datum/outfit/job/cook/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```

Comment:
```suggestion
/datum/outfit/job/cook/post_equip(mob/living/carbon/human/user, visualsOnly = FALSE)
```

The root of `post_equip()` has the first argument name as `user`, so it's important to make sure that remains consistent throughout all subtype overrides of the proc. Single-letter variables are also heavily discouraged regardless. You would need to replace all instances of `H` with `user` in the proc as well (otherwise it doesn't compile).

Reply:
Hmn. So looking into this deeper there's more mismatches.
`/datum/outfit/job/cook/pre_equip` uses `H`, `/datum/outfit/job/pre_equip` uses `H`, but `/datum/outfit/pre_equip` uses `user`.
Similarly `/datum/outfit/job/post_equip` uses `equipped`, but `/datum/outfit/post_equip` uses `user`.

Reply:
The `H` bit was purely to stick with what the cook document was going with, but yeah this has been updated now.
I'll assume making all the other bits I'm not already changing consistent is broadly better as a separate pr.

Reply:
It would be better as a separate PR, we just need named arguments to be as consistent as possible (like if the root proc was named H i would have let it slide) but we just really need less single letter variables regardless